src/main/java/seedu/address/storage/CommandHistoryStorage.java

```diff
@@ -187,4 +187,19 @@ public static void initHistory() {
     public void setCommandHistoryFilePath(Path path) {
         commandHistoryFilePath = path;
     }
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (!(obj instanceof CommandHistoryStorage)) {
+            return false;
+        }
+        CommandHistoryStorage other = (CommandHistoryStorage) obj;
+
+        return commandHistoryFilePath.equals(other.commandHistoryFilePath)
+                && lines.equals(other.lines)
+                && currentLineNumber == other.currentLineNumber
+                && previousTotalLines == other.previousTotalLines;
+    }
```

Comment:
Same apply to this equals method.

Reply:
`CommandHistoryStorageTest.java` now includes test for the `equals()` method.

See commit b5d203f1110644d35c2fffdfe7694b6168979f2f for more details.